doc/sphinx/Clusters_from_Scratch/active-active.rst

```diff
@@ -34,8 +34,8 @@ Configure the Cluster for the DLM
 #################################
 
 The DLM control daemon needs to run on both nodes, so we'll start by creating a
-resource for it (using the **ocf:pacemaker:controld** resource script), and clone
-it:
+resource for it (using the ``ocf:pacemaker:controld`` resource agent), and
+clone it:
```

Comment:
I think the easy cases are:
* UI elements (menu items, etc.): bold
* Literal command-line input or output: fixed-width

Isolated technical terms (like the resource agent above) are less clear. I've generally leaned to bold, but as long as we're consistent, I don't have a strong preference.

Reply:
> I think the easy cases are:

Agreed and AFAIK I've formatted accordingly.

> Isolated technical terms (like the resource agent above) are less clear. I've generally leaned to bold, but as long as we're consistent, I don't have a strong preference.

Agreed here too. There are plenty of cases where **`bold literals`** would be preferable -- whether to emphasize something or to introduce a new term. Unfortunately, nested inline formatting has been on ReST's to-do list for [over 20 years](https://docutils.sourceforge.io/docs/dev/rst/alternatives.html#nested-inline-markup). I feel their pain.

As it stands, having to pick one... formatting all these keywords (and reviewing them ;) ) is tedious. I didn't want to have to weigh options for each "it could go either way" formatting choice. So I went with literals -- in the vein of, for example, the [Red Hat cluster docs](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/configuring_and_managing_high_availability_clusters/index) (see any example of `IPaddr2` for instance).